### ISSUE TITLE
frontend container throwing error during start

### ISSUE THREAD

**Issue by palamvmw** [open-telemetry/opentelemetry-demo#550]
# Bug Report

Which version of the demo you are using? (please provide either a specific
[commit
hash](https://github.com/open-telemetry/opentelemetry-demo/commits/main)
or a specific
[0.9.6](https://github.com/open-telemetry/opentelemetry-demo/releases)).

## Symptom

Following the instructions [here](https://github.com/open-telemetry/opentelemetry-demo/blob/main/docs/docker_deployment.md) , I tried reaching the url http://localhost:8080/
and saw the following error on console.
```37 verbose stack     at ChildProcess.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/@npmcli/promise-spawn/lib/index.js:63:27)
37 verbose stack     at ChildProcess.emit (node:events:513:28)
37 verbose stack     at maybeClose (node:internal/child_process:1100:16)
37 verbose stack     at Process.ChildProcess._handle.onexit (node:internal/child_process:304:5)
38 verbose pkgid frontend@0.1.0
39 verbose cwd /app
40 verbose Linux 5.10.124-linuxkit
41 verbose node v16.18.0
42 verbose npm  v8.19.2
43 error path /app
44 error command failed
45 error signal SIGKILL
46 error command sh -c -- node --require ./Instrumentation.js server.js
47 verbose exit 1
48 timing npm Completed in 13060ms
49 verbose code 1
50 error A complete log of this run can be found in:
50 error     /home/nextjs/.npm/_logs/2022-11-01T22_19_14_173Z-debug-0.log
```

I expected to see the demo frontend application.

## Reproduce

Follow the instructions [here](https://github.com/open-telemetry/opentelemetry-demo/blob/main/docs/docker_deployment.md)

## Additional Context

Creating the `/app` directory in dockerfile fixed the issue locally for me

**Comment by austinlparker** [comment]
Are you on an Apple Silicon machine, or were you trying to build using `docker compose build`?

**Comment by palamvmw** [comment]
Yes, I am on Apple M1 and build the images locally

**Comment by cartersocha** [comment]
Apple silicon issues have hopefully been solved in #536. Please run docker system prune -a then run a fresh docker compose build. Closing for now pending confirmation.
